### DISCUSSION TITLE
0.80: sensor.coral_inference_speed no longer available?

### DISCUSSION THREAD

**Discussion by Ysbrand** [blakeblackshear/frigate#635] [Ask A Question] [answered]
It looks like sensor.coral_inference_speed is no longer available in 0.80 is that correct?
(Yes, I've updated the integration as well to 1.0.2)

And the TPU is recognized:

```
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.app                    INFO    : Capture process started for Voordeur: 43
frigate.edgetpu                INFO    : TPU found
```

And the debug screen does show it:

```
detector | detection start | inference speed | pid
-- | -- | -- | --
coral | 0 | 14.57 | 39


```

**Comment by blakeblackshear** [comment]
The integration should be creating the sensor. It is on all my systems.

**Comment by Ysbrand** [comment, marked as answer]
The sensor was there, but it was listed as unavailable. Turns out that all the other sensors were listed as unavailable as well (since I upgraded to 0.80).

I tried several things, but in the end only a full removal of frigate add-on as well as the HA frigate integration (followed by a reboot and a reinstall) fixed the issue.
